### DISCUSSION TITLE
Porting to HTML

### DISCUSSION THREAD

**Discussion by olderatriangle** [realDunkin/fnfmegapack#26] [General]
How did you manage to port FNF to web browsers? Is it complex, easy, or sort of in the middle? I have been wanting to know how people manage to do this because I have been thinking of porting some mods that I haven't seen anyone port to html such as Vs Dave and Bambi, Cyber Sensation, Blantados's mods such as Monika Full Week Rebooted (Senpai Edition), the BETADCIUs and so many more. Is it kind of hard to compile the code into one big JavaScript file or do you have some sort of converter for it?

**Comment by realDunkin** [comment]
Its pretty simple, just compile the game as usual, but instead of windows do `lime test html5` and then it'll compile the game as html5.
There are some mods that'll take more then that since something's aren't compatible for html5
After you compiled the game just put it on GitHub and turn on GitHub pages!

**Comment by ChickenNuggets-is-cool** [comment]
can you webport fnf on a chromebook ?
